### DISCUSSION TITLE
OpenTelemetry exporting

### DISCUSSION THREAD

**Discussion by marcuskohlberg** [encoredev/encore#1033] [Suggestions]
OTEL exporting has come up a few times, so creating a discussion where folk can weigh in on how they expect to use something like this.

Do you expect to export traces to e.g. DataDog or what's your use case?

**Comment by danhawkins** [comment]
I'm using Tempo and pushing out to there, so for me that would be the most useful

**Comment by macleodmac** [comment]
We use Axiom - our use case for wanting this external from Encore is because we need better searching capabilities - Encore's tracing is great for low volume/local development, but once you want to do more complex filtering or aggregation over time, it leaves us wanting.

**Reply by marcuskohlberg** [reply]
Hey @macleodmac, is this still your preferred mode of operation now that the new trace explorer has been released?

**Reply by macleodmac** [reply]
It is, additionally to this the reason Encore tracing falls down is that there's no support for custom spans - as soon as you diverge away from the Encore operating model (everything being an endpoint) then you lose all visibility.

**Comment by partounian** [comment]
I would love to export OpenTel data to Honeycomb.io

**Comment by AlexSzlavik** [comment]
I'll pile on, years later, that OTEL export would be amazing. I've hand rolled an implementation but it's awkward at best
